measurements/bq/doc/REFERENCE.md

```diff
@@ -0,0 +1,99 @@
+## measurements
+
+<div class="badge core"></div>
+
+### ST_ANGLE
+
+{{% bannerNote type="code" %}}
+measurements.ST_ANGLE(startPoint, midPoint, endPoint, mercator)
+{{%/ bannerNote %}}
+
+**Description**
+
+Finds the angle formed by two adjacent segments defined by 3 points. The result will be the (positive clockwise) angle with origin on the startPoint-midPoint segment, or its explementary angle if required. https://github.com/Turfjs/turf/tree/master/packages/turf-angle
+
+* `startPoint`: `GEOGRAPHY` Start Point Coordinates.
```

Comment:
We, including myself, should be consistent with starting the parameter description with upper or lower case. It should be consistent at least in the same `REFERENCE` file. For instance:

### ST_ANGLE
* `startPoint`: `GEOGRAPHY` Start Point Coordinates.

### ST_AZIMUTH
* `startPoint`: `GEOGRAPHY` starting Point.